code/SQLite3Query.php

```diff
@@ -72,13 +63,10 @@ public function numRecords()
         return $c;
     }
 
-    #[\ReturnTypeWillChange]
-    public function getIterator()
+    public function getIterator(): Traversable
     {
-        if ($data = $this->handle->fetchArray(SQLITE3_ASSOC)) {
-            return $data;
-        } else {
-            return false;
+        while ($data = $this->handle->fetchArray(SQLITE3_ASSOC)) {
```

Comment:
This was initially only returning the first row. I swap it to a `while` statement so it would iterate over the data.